.circleci/config.yml

```diff
@@ -58,6 +59,3 @@ workflows:
           filters:
             tags:
               only: /^v.*/
-            branches:
-              ignore: /.*/
```

Comment:
Maybe its actually this that's causing CircleCI to try to run the deploy step for PRs as well as tags

Reply:
Hmm, on the documentation I've read that if you put an only filter that will disable others, but maybe it only works for the same group like tags and branches.

Reply:
I might be wrong. Maybe you can try putting it back in just to test if circleci skips the deploy step

Reply:
You can check out the relevant documentation here https://circleci.com/docs/2.0/configuration-reference/#jobs-1. In summary: ignore is necessary for skipping the job in a branch build 😀